services/cards/src/Constants.tsx

```diff
@@ -1,3 +1,63 @@
 export const MAX_COUNTER_HEALTH = 30;
 export const MAX_ADVENTURER_HEALTH = 12;
 export const POKER_CARDS_PER_LETTER_PAGE = 9;
+
+export interface CardSource {
+  name: string;
+  key: string;
+  sheets: {[key: string]: string};
+}
+
+export const SHEETS: CardSource[] = [
+  {
+    name: 'Expedition',
+    key: '2PACX-1vQ0Hx1RcDqkc3pn7ZyKEiqdDXm4miEgDeozJ9amJrxz2QzAH8AOaf6rqaLJL4G3fcqCDIYDKdDqsuax',
+    sheets: {
+      Translations: '2111992856',
+      Helper: '987926921',
+      Adventurer: '558719643',
+      Ability: '0',
```

Comment:
Should this be 0? We should have an abilities sheet for the base game (and for the future below)

Reply:
0 is the ID of the Ability sheet. Added a clarifying comment